go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2
 	github.com/eapache/go-resiliency v1.2.0 // indirect
 	github.com/emirpasic/gods v0.0.0-20190624094223-e689965507ab
+	github.com/facebookgo/clock v0.0.0-20150410010913-600d898af40a // indirect
```

Comment:
My guess, if you use go 1.13 this line will not be included

Reply:
thanks :)